src/services/crab-api.service.ts

```diff
@@ -120,29 +120,42 @@ export class CrabApiService extends HttpService {
   }
 
   async getGemeentenPerGewest(niscode: Niscode): Promise<IGemeente[]> {
-    return (await this.get<IGemeente[]>(`adressenregister/gewesten/${niscode}/gemeenten`, { baseURL: this.API_URL }))
-      .data;
+    return (
+      await this.get<IGemeente[]>(`adressenregister/gewesten/${niscode}/gemeenten?status=inGebruik`, {
```

Comment:
Kunt ge de status=inGebruik niet meegeven met de params optie ipv in de base url?